app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,4 @@
+class PagesController < ApplicationController
+  def home
+  end
+end
```

Comment:
Missing new line. Github complaining

Reply:
If you go to `Files changed` section you will see the complain:
<img width="521" alt="image" src="https://user-images.githubusercontent.com/38741141/199871312-07497e4e-c468-44be-b079-7bef2963fb74.png">

Reply:
Fixed